### DISCUSSION TITLE
The error message in custom validator  is not working!

### DISCUSSION THREAD

**Discussion by fellz** [vueform/vueform#235] [Questions] [answered]
Hi 
if i use custom validator and use custom error message it doesn't work!

```
const uppercase = class extends Validator {
  get msg() {
    return 'The text must only contain uppercase letters'
  }

  check(value) {
    return /^[A-Z]*$/.test(value)
  }
}
```

Here is reproduction
https://stackblitz.com/edit/github-yhqznh-zosmhq?file=src%2FApp.vue
Is this a bug ?

**Comment by adamberecz** [comment, marked as answer]
The sandbox was using an earlier version of Vueform. It should now be fixed, please try again with a new sandbox instance: https://stackblitz.com/github/vueform/sandbox

(Working version: https://stackblitz.com/edit/github-urdz63-vn6fce?file=src%2FApp.vue)
